src/components/Message/MessageCommerce.tsx

```diff
@@ -73,7 +74,7 @@ const MessageCommerceWithContext = <
     return <MessageDeleted message={message} />;
   }
 
-  if (message.customType === 'message.date') {
+  if (message.customType === CUSTOM_MESSAGE.date) {
```

Comment:
This does not read very well - you compare `type` to `date`. Should not the constant be `CUSTOM_MESSAGE_TYPE`?

Reply:
I was considering, whether to use `_TYPE ` or not and originally decided to go with the shorter version. But after your feedback I will change it to `CUSTOM_MESSAGE_TYPE`.